webpack.config.js

```diff
@@ -248,14 +248,17 @@ const info = {
 
 process.traceDeprecation = true;
 
+const childProcess = require('child_process');
+const path = require("path");
+const fs = require("fs");
+
 const webpack = require("webpack");
+
 const copy = require("copy-webpack-plugin");
-const childProcess = require('child_process');
 const html = require('html-webpack-plugin');
 const miniCssExtractPlugin = require('mini-css-extract-plugin');
-const path = require("path");
-const fs = require("fs");
 const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin');
```

Comment:
Note: This is mostly noise. I wanted to sort the plugins into a separate block, and keep the "standard node library" modules at the top.